setup.py

```diff
@@ -59,10 +60,19 @@
     pyarrow_version = tuple(int(x) for x in pyarrow.__version__.split("."))
     extensions = cythonize(
         [
+            # vendored arrow iterator
```

Comment:
Why is this vendored?

Reply:
this is the original arrow implementation, we vendored all the arrow binaries to implement our own converter, and make  our connector self-contained (not need to install extra dependency for arrow converter implementation)